R/eurasianphonology.feature.R

```diff
@@ -0,0 +1,22 @@
+#' Download data from the database of Eurasian phonological inventories
+#'
+#' This function open downloaded data from the database of Eurasian phonological inventories (\url{http://eurasianphonology.info}) and changes language names to the names from lingtypology database. You need the internet connection.
```

Comment:
opens

Reply:
And are names for real changed? I'd expect them still to be present.

Reply:
There are two columns. One contains the original names from the eurasian phonology database and the other is consisted of changed names.